pkg/handler/repositories.go

```diff
@@ -193,6 +195,14 @@ func (rh *RepositoryHandler) bulkCreateRepositories(c echo.Context) error {
 		return ce.NewErrorResponseFromError("Error creating repository", errs...)
 	}
 
+	// Send notifications
+	mappedValues := []repositories.Repositories{}
+	for i := 0; i < len(responses); i++ {
+		mappedValues = append(mappedValues, notifications.MapRepositoryResponse(responses[i]))
+	}
+
+	notifications.SendNotification(orgID, notifications.RepositoryCreated, mappedValues)
```

Comment:
should we add the  non-bulk create notification as part of this, or in a followup?

Reply:
We can, as a follow up.